.git-blame-ignore-revs

```diff
@@ -0,0 +1,6 @@
+# Format with Black
+53a7e3500437a9fd5826bc04758f7116bd7e52dc
+# Format the C code with ClangFormat
+46b7e86bab79450ec0a2866c6c0c679afb659d17
+# Flake8
+8de95676e0fd89f2326b3953488ab66ff29cd2d0
```

Comment:
```suggestion
# Flake8
8de95676e0fd89f2326b3953488ab66ff29cd2d0
# Format with Black
53a7e3500437a9fd5826bc04758f7116bd7e52dc
# Format the C code with ClangFormat
46b7e86bab79450ec0a2866c6c0c679afb659d17
```
It would seem good to have some sort of order, so my suggestion would be from oldest to newest.